- Add missing include path for hidapi
- Instead of statically linking hid-libusb.o in, link with -lhidapi-libusb

--- src-orig/Makefile
+++ src/Makefile
@@ -25,7 +25,8 @@
 	-I./DLL430_v3/src/TI/DLL430/EnergyTrace_TSPA \
 	-I./Bios/include \
 	-I./ThirdParty/include \
-	-I./ThirdParty/BSL430_DLL
+	-I./ThirdParty/BSL430_DLL \
+	-I/usr/include/hidapi
 
 
 LIBS :=
@@ -49,14 +50,8 @@
 	OUTPUT := libmsp430.so
 	DEFINES += -DUNIX
 
-	ifdef STATIC
-	STATIC_LIBS += -lusb-1.0
-	else
-	LIBS += -lusb-1.0
-	endif
+	LIBS += -lusb-1.0 -lrt -lpthread -lhidapi-libusb
 
-	LIBS += -lusb-1.0 -lrt -lpthread
-
 	ifdef BOOST_DIR
 	INCLUDES += -I$(BOOST_DIR)
 	LIBDIRS += -L$(BOOST_DIR)/stage/lib
@@ -66,7 +61,6 @@
 	BSTATIC := -Wl,-Bstatic
 	BDYNAMIC := -Wl,-Bdynamic
 
-	HIDOBJ := $(LIBTHIRD)/hid-libusb.o
 else
 	CXX:= clang++
 
--- src-orig/ThirdParty/BSL430_DLL/Makefile
+++ src/ThirdParty/BSL430_DLL/Makefile
@@ -21,7 +21,8 @@
 	-I./BSL430_DLL/Utility_Classes \
 	-I./BSL430_DLL/Physical_Interfaces \
 	-I./BSL430_DLL/Packet_Handlers \
-	-I./BSL430_DLL/Connections
+	-I./BSL430_DLL/Connections \
+	-I/usr/include/hidapi
 
 SRC := \
 	./BSL430_DLL/MSPBSL_Factory.cpp \
